src/components/common/Actions.vue

```diff
@@ -350,7 +342,7 @@ export default class Actions extends Vue {
   /** Label for the Next button. */
   get nextButtonLabel (): string {
     const nextStep = this.next()
-    return nextStep ? nextStep.text.replace('\n', ' ') : ''
+    return nextStep ? nextStep.text.replaceAll('\n', ' ') : ''
```

Comment:
Another fix -- replace() only changes the first match, so we need to use replaceAll().